.github/CHANGELOG.md

```diff
@@ -53,6 +55,7 @@ Joseph Lee,
 Anton Naim Ibrahim,
 Luis Alfredo Nuñez Meneses,
 Andrija Paurevic,
+Runor Agbaire
```

Comment:
It's in alphabetical order based on surname :) Feel free to add your name to the top of the list.